packages/dashboard/src/customization/widgets/barChart/component.tsx

```diff
@@ -47,7 +48,7 @@ const BarChartWidgetComponent: React.FC<BarChartWidget> = (widget) => {
       <BarChart
         chartSize={size}
         key={key}
-        queries={queries}
+        queries={queries.filter((q) => 'build' in q) as TimeSeriesDataQuery[]}
```

Comment:
This goes for all of the widget query/queries fields: I see you updated the interface to accept `TimeSeriesDataQuery | AlarmDataQuery`, so why do we need to filter them here? Shouldn't we be filtering the queries in the widget component itself when it passes them to `useTimesSeriesData` and `useAlarms`?